tap_clickup/tap.py

```diff
@@ -47,6 +47,15 @@ class TapClickUp(Tap):
         th.Property(
             "api_token", th.StringType, required=True, description="Example: 'pk_12345"
         ),
+        th.Property(
+            "workspace_id", th.IntegerType, required=False, description="Example: '20214542" # fetches the data for workspace_id
+        ),
+        th.Property(
+            "spaces_id", th.StringType, required=False, description="Example: '[45215477,4547547]" # fetches the data for workspace_id
```

Comment:
This should be an ArrayType see https://sdk.meltano.com/en/latest/classes/typing/singer_sdk.typing.ArrayType.html